client/src/pages/Login.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useContext } from 'react'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core/styles';
+import * as imageURL from '../images/login-photo.png'
+import { Link } from 'react-router-dom'
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+      width: '25ch',
+      length: '400ch',
+      display: 'flex',
+      flexDirection: 'column',
+      marginBottom: '30px'
+    }
+  },
+}));
+
+const ContinueButton = withStyles({
+  root: {
+    background: 'linear-gradient(45deg, rgba(73, 94, 203, .8) 30%, rgba(73, 145, 203, .8) 90%)',
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+    boxShadow: '0 3px 5px 2px rgba(125, 123, 135, .3)',
+    marginLeft: '10px'
+  },
+})(Button);
+
+const SignupButton = withStyles({
+  root: {
+    background: 'linear-gradient(45deg, rgba(105, 105, 105, .8) 30%, rgba(169, 169, 169, .8) 90%)',
+    color: 'white',
+    height: '5ch',
+    padding: '0 30px',
+    boxShadow: '0 3px 5px 2px rgba(125, 123, 135, .3)',
+    marginLeft: '10px'
+  },
+})(Button);
+
+const Login = () => {
+
+  const classes = useStyles();
+
+  const [formData, setFormData] = useState({
+    email: '',
+    password: ''
+  });
+
+  const { firstName, password } = formData;
+
+  const onChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  }
+  const onSubmit = async (e) => {
+    e.preventDefault();
+  };
+
+  const passwordIsValid = () => {
+    if (password.length >= 6) {
+      return true
+    } else {
+      return false
+    }
+  }
+
+  const areAllFieldsCompleted = () => {
+    if (firstName.length === 0 || firstName.length === 0 || firstName.length === 0 || firstName.length === 0) {
+      return false
+    } else {
+      return true
+    }
+  }
+  return (
+    <div className='login-sign-up-container'>
+      <img className='login-image' src={imageURL}></img>
```

Comment:
I noticed a couple things on the styling:
1. The app is not fullscreen - the image is more of floating in the page, as opposed to covering the full left side of the screen
2. You are using floats, etc. Instead, you should be using the material-UI Grid system. This will allow you to also build a sign up and login page that is more responsive